src/viur/core/bones/phone.py

```diff
@@ -0,0 +1,119 @@
+import re
+from typing import Optional, Pattern, Tuple, List, Any, Dict
+
+from viur.core.bones.string import StringBone
+from viur.core.bones.base import ReadFromClientError, ReadFromClientErrorSeverity
+
+
+class PhoneBone(StringBone):
+    """
+    The PhoneBone class is designed to store validated phone/fax numbers in configurable formats.
+    This class provides a number validation method, ensuring that the given phone/fax number conforms to the
+    required/configured format and structure.
+    """
+    type: str = "str.phone"
+    """
+    A string representing the type of the bone, in this case "str.phone".
+    """
+    def __init__(
+        self,
```

Comment:
Parameters for bones are always kwarg-only, therefor you need to specify it. See https://peps.python.org/pep-3102/ for details.

```suggestion
        self,
        *,
```